### DISCUSSION TITLE
Welcome to SeleniumBase Discussions!

### DISCUSSION THREAD

**Discussion by mdmintz** [seleniumbase/SeleniumBase#750] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by kennedylunar** [comment]
👋🏽 Hello, kenn here. Leaning test automation during and after work, still a newbie in terms of doing framework and advance test automation. 😃

**Reply by mdmintz** [reply]
Welcome! There are lots of ReadMe docs to help you get started. There are also some tutorial videos located on YouTube: https://www.youtube.com/watch?v=Sjzq9kU5kOw&list=PLp9uKicxkBc5UIlGi2BuE3aWC7JyXpD3m
